### ISSUE TITLE
Demonstrate the effects of the perturbed polybeam through imaging

### ISSUE THREAD

**Issue by steven-murray** [HERA-Team/hera_sim#327]
Maybe @r-pascua wants to update this with details...

**Comment by r-pascua** [comment]
Right. So the current polybeam tutorial notebook shows a gridded version of the beam and demonstrates its effects on the data by plotting an autocorrelation amplitude as a function of beam rotation angle. I was thinking that it would be nice to have some comparison images: put something on the sky (like the HERA text from the `pyuvsim` reference sims), simulate visibilities with various perturbed polybeams, then image the simulations and show how they compare to images made with the unperturbed beam simulations. This is by no means a priority, but I think it could be helpful from a pedagogical perspective.
